controllers/complianceeventsapi/utils.go

```diff
@@ -0,0 +1,11 @@
+package complianceeventsapi
+
+func ContainRules(rules []string, input string) bool {
```

Comment:
I wanted to use `slice.contains` but our go version is 1.20

Reply:
Just bumped to 1.21, does that help?